test_scripts/API/GetWayPoints/common_get_way_points.lua

```diff
@@ -0,0 +1,273 @@
+---------------------------------------------------------------------------------------------------
+-- GetWayPoints common module
+---------------------------------------------------------------------------------------------------
+--[[ General configuration parameters ]]
+config.deviceMAC = "12ca17b49af2289436f303e0166030a21e525d266e209267433801a8fd4071a0"
+config.defaultProtocolVersion = 2
+config.ValidateSchema = false
```

Comment:
@LitvinenkoIra Please remove this line ("config.ValidateSchema") since it was required for RC only due to the fact that initially RC RPCs were not defined in APIs